Gemfile

```diff
@@ -12,7 +12,7 @@ end
 
 gem 'artifactory'
 gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '0.15.12')
-gem 'packaging', :github => 'puppetlabs/packaging', branch: '1.0.x'
+gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99.7')
```

Comment:
You will need to update the `vanagon_location_for` method above to work with this. There should be prior-art in other repos.
Also, while 0.99.7 is fine I think, I'd use 0.99.8 since that's the latest.

Reply:
Fixed.